Example/CollectionViewToolsExample/MainViewController.swift

```diff
@@ -111,6 +111,17 @@ class MainViewController: UIViewController {
             detailViewController.image = image
             self?.navigationController?.pushViewController(detailViewController, animated: true)
         }
+        cellItem.itemDidSelectHandler = { [weak cellItem] _ in
+            weak var imageView = (cellItem?.cell as? ImageCollectionViewCell)?.imageView
+            UIView.animate(withDuration: 0.2, animations: {
+                imageView?.alpha = 0.5
+            }, completion: { _ in
```

Comment:
It is seems like you are trying to implement a sort of highlighting. If it is then I think that there are a better way to do it 🤔

Reply:
it is just an example how it is possible to access to the cell via cellItem